### PR TITLE
Renamed : kebab-case filename to snake_case

### PR DESCRIPTION
**Pull request by Srishti013**
# Description:

I have changed the name of kebab-case files to snake_case.

Fixes # (issue no.) 
fixes[#165]
## Type of change:
- [x] Bug fix (non-breaking change which fixes an issue)

 
# Checklist:
- [x] My changes generate no new warnings.

https://user-images.githubusercontent.com/67790510/118239065-ee4cf300-b4b6-11eb-90de-f59d9f9f0846.mov

### CODE REVIEW

**Comment by Aryamanz29** [comment]
Hii 👋 @Srishti013 
PR looks good, but you have to do some minors changes 📝
- Change PR title ( should be more descriptive)🎯
-  I think you may forget issue 4th point
     - **Note: Do changes in the codebase as well if they link into the project**. 
- Please change these rename file names in the codebase as well like in the `views.py render` function,` HTML` files link `JS files`, etc📜
- Also don't forget to add a working `GIF/Video` in the PR description so that we can verify that your work doesn't break any existing feature 🎞

File: templates/forgot_password.html [hunk 1]
```diff
@@ -80,4 +80,4 @@ <h2>Reset Password</h2>
   <script src="{% static 'js/reset-pwd.js' %}"></script>
```
Comment (Aryamanz29) [inline templates/forgot_password.html:80]: Rename `js/reset-pwd.js`

**Review by Aryamanz29 — CHANGES_REQUESTED** [review]

**Review by Aryamanz29 — APPROVED** [review]

**Comment by Aryamanz29** [comment]
@Srishti013 Nice work 🎉
- Now squash your commits (combine all your commits into `one  single commit`) for ref :https://levelup.gitconnected.com/how-to-squash-git-commits-9a095c1bc1fc
- Add GIF/Video in PR description.

**Comment by Srishti013** [comment]
Sure

**Comment by SaiSudhaV** [comment]
> Hii 👋 @Srishti013
> PR looks good, but you have to do some minors changes 📝
> 
> * Change PR title ( should be more descriptive)🎯
> * I think you may forget issue 4th point👇
>   
>   * **Note: Do changes in the codebase as well if they link into the project**.
> * Please change these rename file names in the codebase as well like in the `views.py render` function,` HTML` files link `JS files`, etc📜
> * Also don't forget to add a working `GIF/Video` in the PR description so that we can verify that your work doesn't break any existing feature 🎞
> 
> @sruti2024 @SaiSudhaV What do you think?

Yeah you're right 
@Srishti013 Please do following things
 - add working demo video / GIF
 - Link the issue

**Comment by Srishti013** [comment]
> > Hii 👋 @Srishti013
> > PR looks good, but you have to do some minors changes 📝
> > 
> > * Change PR title ( should be more descriptive)🎯
> > * I think you may forget issue 4th point👇
> >   
> >   * **Note: Do changes in the codebase as well if they link into the project**.
> > * Please change these rename file names in the codebase as well like in the `views.py render` function,` HTML` files link `JS files`, etc📜
> > * Also don't forget to add a working `GIF/Video` in the PR description so that we can verify that your work doesn't break any existing feature 🎞
> > 
> > @sruti2024 @SaiSudhaV What do you think?
> 
> Yeah you're right
> @Srishti013 Please do following things
> 
> * add working demo video / GIF
> * Link the PR with the issue you're working on

done

**Comment by Aryamanz29** [comment]
> > > Hii 👋 @Srishti013
> > > PR looks good, but you have to do some minors changes 📝
> > > 
> > > * Change PR title ( should be more descriptive)🎯
> > > * I think you may forget issue 4th point👇
> > >   
> > >   * **Note: Do changes in the codebase as well if they link into the project**.
> > > * Please change these rename file names in the codebase as well like in the `views.py render` function,` HTML` files link `JS files`, etc📜
> > > * Also don't forget to add a working `GIF/Video` in the PR description so that we can verify that your work doesn't break any existing feature 🎞
> > > 
> > > @sruti2024 @SaiSudhaV What do you think?
> > 
> > 
> > Yeah you're right
> > @Srishti013 Please do following things
> > 
> > * add working demo video / GIF
> > * Link the PR with the issue you're working on
> 
> done

@Srishti013 Please squash your commits.

**Pull request merged**